src/openrct2/ride/Vehicle.h

```diff
@@ -155,6 +156,8 @@ struct rct_vehicle_info
 
 struct SoundIdVolume;
 
+using VehicleSoundParamList = FixedVector<OpenRCT2::Audio::VehicleSoundParams, OpenRCT2::Audio::MaxVehicleSounds>;
```

Comment:
Do we need to use the using here could we forward declare the class then can avoid adding the header